HIP/hip-smart-contract-besu.md

```diff
@@ -0,0 +1,244 @@
+- hip:
+- title: Migrate Smart Contract Service EVM to Hyperledger Besu EVM
+- author: Daniel Ivanov (daniel@limechain.tech), Danno Ferrin (
+  danno.ferrin@hedera.com)
+- type: Standards
+- category: Services
+- status: Draft
+- created: 16 September 2021
+- discussions-to:
```

Comment:
Please link to this Discussion thread: https://github.com/hashgraph/hedera-improvement-proposal/discussions/140

Reply:
done